common/tools/mcp-server/config/rush-project.json

```diff
@@ -0,0 +1,3 @@
+{
+  "extends": "../../../../common/config/rush-project.json",
```

Comment:
Not sure what this file does, but now that I search for it I see we have one in a few different packages (all of Core, Identity, and some random client libraries). Just for my own learning, what's it for?